frontend/src/app/services/storage-handler.service.ts

```diff
@@ -6,100 +6,132 @@ import { Injectable } from '@angular/core';
 import { Storage } from '@ionic/storage';
 
 import { FormsData } from '../model/forms-data.model';
+import { RatingForm } from '../rating/model/rating-form.model';
 
 @Injectable({
   providedIn: 'root',
 })
 export class StorageHandlerService {
-  constructor(private storage: Storage) {}
+  applicantDetailsDb: Storage;
+  apllicantRatingsDb: Storage;
+  commonPropertiesDb: Storage;
+
+  constructor() {
+    this.applicantDetailsDb = new Storage({
+      name: 'digitact',
+      storeName: 'applicants-details',
+      driverOrder: ['indexeddb', 'localstorage'],
+    });
+    this.apllicantRatingsDb = new Storage({
+      name: 'digitact',
+      storeName: 'applicants-rating',
+      driverOrder: ['indexeddb', 'localstorage'],
+    });
+    this.commonPropertiesDb = new Storage({
+      name: 'digitact',
+      storeName: 'common-properties',
+      driverOrder: ['indexeddb', 'localstorage'],
+    });
```

Comment:
I am happy that it worked like that. :) We definitely shouldn't forget to add the sqlite storage as well

Reply:
SQLite is next priority.